src/main/java/jp/ac/keio/bio/fun/xitosbml/cui/CuiRun.java

```diff
@@ -48,5 +49,5 @@ public static void main(String... args) {
  * @SuppressWarnings("deprecation") public static void main(String... args) {
  * CommandLine.run(new CuiRun(), System.err, args); }
  * 
- * }
- */
\ No newline at end of file
+ * } <<<<<<< HEAD
```

Comment:
You should delete `<<<HEAD` part

Reply:
Sorry! I had not been careful. Removed <<<Head part.